waku/common/validators.go

```diff
@@ -0,0 +1,13 @@
+package common
+
+// ValidateDataIntegrity returns false if the data have the wrong or contains all zeros,
+// which is the simplest and the most common bug.
+func ValidateDataIntegrity(k []byte, expectedSize int) bool {
+	if len(k) != expectedSize {
+		return false
+	}
+	if expectedSize > 3 && ContainsOnlyZeros(k) {
+		return false
+	}
+	return true
+}
```

Comment:
This feels like a case of https://github.com/status-im/status-go/pull/1947/files#r415707643, but there is a known pattern of placing validation functions into `validators.go` so I think this case makes sense to me.